pages/common/find.md

```diff
@@ -19,9 +19,9 @@
 
 `find {{root_path}} -name '{{*.py}}' -not -path '{{*/site-packages/*}}'`
 
-- Find files matching a given size range:
+- Find files matching a given size range excluding subdirectories:
```

Comment:
```suggestion
- Find files matching a given size range, excluding subdirectories:
```
To be consistent with the above description.

Reply:
[fc92c71](https://github.com/tldr-pages/tldr/pull/8719/commits/fc92c71753157c7250524c7f56ccc4e5587301db) includes this suggestion. Thank you.